db/migrations/0005.canvas_usage_table.py

```diff
@@ -0,0 +1,17 @@
+#
+# file: migrations/0005.canvas_usage_table.py
+#
+from yoyo import step
+
+__depends__ = {'0004.add_sis_id'}
+step('''
+    CREATE TABLE IF NOT EXISTS canvas_usage
+    (
+        canvas_id INTEGER NOT NULL,
```

Comment:
Is this `canvas_id` for a user? I think `user_id` might be clearer then? It might also make sense to add an auto-incremented primary key.

Reply:
This is not at the user level but at the course level. This is course_id actually

Reply:
Oh, gotcha. So maybe `course_id`?

Reply:
Maybe I will change the table name to `course_canvas_usage` that will be more detailed to say. maybe in future we will track student-level who knows